fedimint-rocksdb/src/lib.rs

```diff
@@ -103,17 +107,127 @@ impl<'a> IDatabaseTransaction<'a> for RocksDbTransaction<'a> {
 }
 
 #[cfg(test)]
-mod tests {
+mod fedimint_rocksdb_tests {
     use crate::RocksDb;
 
     #[test_log::test]
-    fn test_basic_dbtx_rw() {
+    fn test_dbtx_insert_elements() {
+        let path = tempfile::Builder::new()
```

Comment:
This + `Rocksdb::open` could be it's own function, then these tests could be one-liners.